onesignal-admin.php

```diff
@@ -189,7 +190,7 @@ public static function on_save_post($post_id, $post, $updated)
         /* Some WordPress environments seem to be inconsistent about whether on_save_post is called before transition_post_status
            * Check flag in case we just sent a notification for this post (this on_save_post is called after a successful send)
           */
-        $just_sent_notification = (get_post_meta($post_id, 'onesignal_notification_already_sent') === true);
+        $just_sent_notification = (get_post_meta($post_id, 'onesignal_notification_already_sent', true) === true);
```

Comment:
Double check that `get_post_meta($post_id, 'onesignal_notification_already_sent', true)` returns a `boolean`